pkg/antctl/command_definition.go

```diff
@@ -295,7 +295,9 @@ func (cd *commandDefinition) applySubCommandToRoot(root *cobra.Command, client A
 	if groupCommand, ok := groupCommands[cd.commandGroup]; ok {
 		groupCommand.AddCommand(cmd)
 	} else {
-		root.AddCommand(cmd)
+		if cmd.Use != "log-level [level]" || (cmd.Use == "log-level [level]" && runtime.InPod) {
```

Comment:
do we need ` [level]` here?

Reply:
yes because when I printed cmd.Use value while debugging it was "log-level [level]", and using only "log-level" didn't work.

Reply:
Looks like it's `applyFlagsToCommand` which adds flags to `cmd.Use` based on original command list definition.
https://github.com/antrea-io/antrea/blob/ecece4256bafa9900f25b580dd503260a909cc1d/pkg/antctl/command_definition.go#L635

Reply:
yes

Reply:
Could we add some comments to explain why we add the check for log-level?

Reply:
s/flowAggregator/FlowAggregator/